scripts/integration_test_contracts.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+
+# sleep
+# This sleep is the sum of the time when docker is pulled, started,
+# and the height of the node increases to 1 or more.
+sleep 30
+
+# send transaction to node
+cur_path=`pwd`
+chain_dir=${cur_path}"/finschia/template/.finschia"
+artifacts_path=${cur_path}"/../artifacts/"
+
+echo "hogehoge"
+echo ${artifacts_path}
+echo ${chain_dir}
+
+#*** store collection contract ***
+# store contract
+fnsad tx wasm store ${artifacts_path}/collection.wasm --node http://localhost:26658 --home=${chain_dir} --from link146asaycmtydq45kxc8evntqfgepagygelel00h --chain-id simd-testing --keyring-backend test --gas 10000000  -b block -o json -y > tmp.json
+
+# confirm a result of `store`
+key=$(cat ./tmp.json | jq '.logs[] | .events' | jq '.[0]' | jq '.attributes[2]' | jq '.key')
+value=$(cat ./tmp.json | jq '.logs[] | .events' | jq '.[0]' | jq '.attributes[2]' | jq '.value')
+expected_key="\"module\""
+expected_val="\"wasm\""
+
+test "${expected_key}" = "${key}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
+test "${expected_val}" = "${value}" ; if [ $? -eq 0 ]; then echo "ok"; else echo "ng"; fi
```

Comment:
```suggestion
if [[ "${expected_key}" = "${key}" ]]; then echo "ok"; else echo "ng"; fi
if [[ "${expected_val}" = "${value}" ]]; then echo "ok"; else echo "ng"; fi
```

And, I doubt are these needed, too. At least, else should stop executing this script.